narrow_valley.py

```diff
@@ -0,0 +1,647 @@
+import time
+import random
+
+#<------------------------------------------------------------ Basic Functions ------------------------------------------------------------>    
+
+def coin_flip():
+	return random.choice(['heads', 'tails'])
+
+def die_roll():
+	return random.randint(1, 6)
+
+def time_print(string):
+    print(string)
+    time.sleep(2)
+
+def time_print_loop(lst):
+    for i in range(len(lst)):
+        time_print(lst[i])
+
+def time_print_img(lst):
+    for i in range(len(lst)):
+        print(lst[i])
+        time.sleep(.5)
+
+def valid_input(promt, option1, option2):
+    while True:
+        response = input(promt).lower()
+        if option1 in response:
+            return option1
+        if option2 in response:
+            return option2
+        else:
+            time_print("I don't understand.")
+    return response
+
+def contine_on():
```

Comment:
Style suggestion: Did you mean `continue_on`?

Reply:
Furthermore, if this function is just meant for the user to "continue" the flow of the story, does it matter what the input is? For example, could it just say "Press Enter to continue."? 

In that case you don't need to check the response, and you can just break it right after this line I think:

`response = input("Enter (1) to continue.\n").lower()`